text/0146-deflationary-fee-proposal.md

```diff
@@ -0,0 +1,47 @@
+# RFC-0146: Deflationary Transaction Fee Model for Relay- and System Chains
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 20th May 2025                                                            |
+| **Description** | This RFC proposes burning 80% of transaction fees on the Relay Chain and System Chains, adding to the existing deflationary capacity.     |
+| **Authors**     | Jonas Gehrlein                                                           |
+
+## Summary
+
+This RFC proposes **burning 80% of transaction fees** accrued on Polkadot’s **Relay Chain** and, more significantly, on all **System Chains**. The remaining 20% would continue to incentivize Validators (on the Relay Chain) and Collators (on System Chains) for including transactions.
```

Comment:
Is there any reasoning behind this specific 80:20 ratio?

Reply:
Yes, it would leave validator rewards untouched and yield a unified ratio across all System Chains.

Reply:
I added this part excplicitly.